Makefile

```diff
@@ -3,12 +3,12 @@ all: clean ocs-operator ocs-must-gather
 .PHONY: ocs-operator
 ocs-operator:
 	@echo "Building ocs-operator image"
-	operator-sdk build quay.io/openshift/ocs-operator
+	operator-sdk build quay.io/ocs-dev/ocs-operator
```

Comment:
This is a bit of nit-picking, but the changes that adapt the image names on quay.io are not related to the topic of the PR. I suggest moving them to a separate PR that clearly reflects this change.

Reply:
I have submitter #20 to update the makefile with these, and few more changes. The Makefile changes can dropped from here.

Reply:
cool dropped.